db/schema.rb

```diff
@@ -10,11 +10,25 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180712020424) do
+ActiveRecord::Schema.define(version: 20190204213636) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "active_admin_comments", force: :cascade do |t|
```

Comment:
@CouchofTomato What's the significance of this table?

Reply:
It's related to https://activeadmin.info/1-general-configuration.html#comments

It's a standard behaviour we can remove if necessary.